vampire_survivors/src/projectile.py

```
@@ -0,0 +1,19 @@
+import pygame
+
+class Projectile(pygame.sprite.Sprite):
+    def __init__(self, pos, direction, groups):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)